examples/NonArduino/Pico/main.cpp

```diff
@@ -44,7 +44,7 @@
 #include "PicoHal.h"
 
 // create a new instance of the HAL class
-PicoHal* hal = new PicoHal(SPI_PORT, SPI_MISO, SPI_MOSI, SPI_SCK);
+PicoHal* hal = new PicoHal(SPI_PORT, SPI_MISO, SPI_MOSI, SPI_SCK, RADIOLIB_NC);
```

Comment:
The need for this new PWM pin is not really explained here, and I'm not sure it is necessary. The only thing it's sued for is configuring it as output, which can be done in the main file. Most users will set this to `RADIOLIB_NC` anyway.

Reply:
I can get behind that logic. That will be removed.